joomla/joomla-cms
# [4.0][codemirror] Add space between attributes

### Summary of Changes
Fix markup. Add space between attributes.


### Testing Instructions
Go to System > Plugins.
Edit `Editor - CodeMirror`.
View page source.
Find `<joomla-editor-codemirror`.
See no space between attributes.


### Actual result
![codemirror](https://user-images.githubusercontent.com/368084/70381769-461c1d80-1905-11ea-8721-6db667459927.jpg)
